config/crd/experimental/policy.networking.k8s.io_adminnetworkpolicies.yaml

```diff
@@ -115,21 +107,14 @@ spec:
                             description: |-
                               NamedPort selects a port on a pod(s) based on name.
 
-                              Support: Extended
-
                               <network-policy-api:experimental>
                             type: string
                           portNumber:
-                            description: |-
-                              Port selects a port on a pod(s) based on number.
-
-                              Support: Core
+                            description: Port selects a port on a pod(s) based on
+                              number.
```

Comment:
Is this valid YAML? (line-wrapped string without any indicator like `|-` or whatever)

Reply:
hmmm just to be sure I can re-generate, I didn't touch these files they were auto-done using `make generated` but let me double check cause there was a conflict on rebasing to master and I had to remove the changes and re-generate it

Reply:
I double checked and it looks correct to me
its normal pattern if the description is only 1 line long: https://github.com/kubernetes-sigs/network-policy-api/blob/563e4ba25a1adf5dc8ec3564e869265aa6169b5c/config/crd/experimental/policy.networking.k8s.io_adminnetworkpolicies.yaml#L263 like here for match expressions

given we had support: core it was appearing as two line formats which is the current case for namedports + experimental which fallson second line.
So if the description is only 1 line long that's how it usually is without the "|-"